### ISSUE TITLE
Does validation set has unseen emoji?

### ISSUE THREAD

**Issue by haofuly** [zhangchuheng123/GrinningFace#2]
Hi, @zhangchuheng123 .
In Section 3, the validation protocol is `Val evaluates the model on the emojis sampled from the validation set.` Does part of these emoji in val set appear in training set, or they are absolutely unseen from the training set. 
Thanks a lot.

**Comment by zhangchuheng123** [comment]
Yes, they are unseen during training (fine-tuning).
